src/csvcubed/cli/inspect/metadataprinter.py

```diff
@@ -67,11 +66,7 @@ class MetadataPrinter:
     This class produces the printables necessary for producing outputs to the CLI.
     """
 
-    data_cube_state: Optional[DataCubeState]
-    code_list_state: Optional[CodeListState]
-    csvw_type: CSVWType
-    csvw_metadata_rdf_graph: rdflib.ConjunctiveGraph
-    csvw_metadata_json_path: Path
+    state: Union[DataCubeState, CodeListState]
```

Comment:
I'm tempted to rename these things `CsvWInspector`, `DataCubeInspector` and `CodeListInspector`. You two got any thoughts on that?